src/api/firebase.js

```diff
@@ -50,6 +58,20 @@ export function getItemData(snapshot) {
 		 */
 		data.id = docRef.id;
 
+		let isDefaultChecked;
```

Comment:
Adding some comments here about this logic would be helpful for others working in this component in the future!